src/workloads/docs/Loader.yml

```diff
@@ -0,0 +1,83 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+
+# More threads requires a larger connection pool.
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 200
+
+Actors:
+  - Name: MultipleCollectionsPerLoaderThread
+    Type: Loader
+    Threads: 10
+    Phases:
+      # If CollectionCount is less than the PhaseConfig Threads, then unexpected outcomes will occur.
+      - Repeat: 1
+        # create 6 collection (CollectionCount / PhaseConfig Threads) per thread over 10 threads.
+        # Thread 0 creates collection 0 to 5
+        # Thread 1 creates collection 6 to 11
+        # etc.
+        Database: hot
+        CollectionCount: 6
+        Threads: 1
+        DocumentCount: 200000
+        BatchSize: 100000
+        Document:
+          a: {^RandomString: { length: 100 }}
+        FindOptions:
+          Hint: a_index # Currently only support the index name.
+          Comment: "Phase 1 loader"
+        # Each thread creates the indexes for each collection it manages.
+        Indexes:
+          - keys: {a: 1}
+            options: {name: "a_index"}
+      - Repeat: 1
```

Comment:
This is another example of unintended and bad behavior, yes?

Reply:
I don't think so, this actually is internally consistent. I assumed this is how it was supposed to work?

Reply:
I expected Threads: 10, and ending up with exactly 6 collections would be the expected case, like in BigUpdate. My expectation is CollectionCount should be the total number of collections added. I think it did that when I first wrote it.  

Before this change https://github.com/mongodb/genny/pull/240 threads was only specified in one place.

Reply:
I was also surprised that CollectionCount is not the number collections created but to be fair I was also surprised that threads must be specified twice (especially if it is the same value). 

I don't want to change the current behavior as LargeScaleModel.yml , LatrgeScaleSerial.yml, OutOfCacheScanner.yml and OutOfCacheSnapShotReads.yml use odd combinations of values.